addons/binding/org.openhab.binding.groheondus/src/main/java/org/openhab/binding/groheondus/internal/handler/GroheOndusAccountHandler.java

```diff
@@ -35,12 +42,20 @@
 @NonNullByDefault
```

Comment:
Could you please add javadoc for this class and at least explain your additions?